package.json

```diff
@@ -149,6 +154,26 @@
       {
         "view": "deepcode.views.analysis",
         "contents": "DeepCode analyzed your code and found no issue! 🎉"
+      },
+      {
+        "view": "deepcode.views.actions",
+        "contents": "You are currently running DeepCode in manual mode.\n[Analyze now](command:deepcode.start)\n[Switch to auto-scan mode](command:deepcode.setmode?%5B%22auto%22%5D)",
+        "when": "deepcode:mode == 'manual'"
+      },
+      {
+        "view": "deepcode.views.actions",
+        "contents": "DeepCode analysis is currently paused.\n[Unpause](command:deepcode.setmode?%5B%22auto%22%5D)",
+        "when": "deepcode:mode == 'paused'"
+      },
+      {
+        "view": "deepcode.views.actions",
+        "contents": "You are currently running DeepCode in throttled mode. It scans your code every 30 minutes if it detects changes in your files.\n[Auto](command:deepcode.setmode?%5B%22auto%22%5D)",
```

Comment:
You are currently running DeepCode in a throttled mode - it scans your code every 30 minutes if it detects changes in your files.